Packages/com.unity.inputsystem/Documentation~/Interactions.md

```diff
@@ -109,23 +132,34 @@ The Input System package comes with a set of basic Interactions you can use. If
 
+The following diagram shows the behavior of the built-in interactions for a simple button press.
+
+![Interaction Diagram](./Images/InteractionsDiagram.png)
+
 ### Default Interaction
 
 If you haven't specifically added an Interaction to a Binding or its Action, the default Interaction applies to the Binding.
 
-[`Value`](Actions.md#value) or [`Button`](Actions.md#button) type Actions have the following behavior:
+[`Value`](Actions.md#value) type Actions have the following behavior:
 
 1. As soon as a bound Control becomes [actuated](Controls.md#control-actuation), the Action goes from `Waiting` to `Started`, and then immediately to `Performed` and back to `Started`. One callback occurs on [`InputAction.started`](../api/UnityEngine.InputSystem.InputAction.html#UnityEngine_InputSystem_InputAction_started), followed by one callback on [`InputAction.performed`](../api/UnityEngine.InputSystem.InputAction.html#UnityEngine_InputSystem_InputAction_performed).
 2. For as long as the bound Control remains actuated, the Action stays in `Started` and triggers `Performed` whenever the value of the Control changes (that is, one call occurs to [`InputAction.performed`](../api/UnityEngine.InputSystem.InputAction.html#UnityEngine_InputSystem_InputAction_performed)).
 3. When the bound Control stops being actuated, the Action goes to `Canceled` and then back to `Waiting`. One call occurs to [`InputAction.canceled`](../api/UnityEngine.InputSystem.InputAction.html#UnityEngine_InputSystem_InputAction_canceled).
 
-[`PassThrough`](Actions.md#pass-through) type Actions have a simpler behavior. The Input System doesn't try to track any Interaction states (which would be meaningless if tracking several Controls separately). Instead, it triggers a `Performed` callback for each value change.
+[`Button`](Actions.md#button) type Actions have the following behavior:
+
+1. As soon as a bound Control becomes [actuated](Controls.md#control-actuation), the Action goes from `Waiting` to `Started`. One callback occurs on [`InputAction.started`](../api/UnityEngine.InputSystem.InputAction.html#UnityEngine_InputSystem_InputAction_started).
+2. If a Control then reaches or exceeds the button press threshold (the default for which is defined in the [input settings](../api/UnityEngine.InputSystem.InputSettings.html#UnityEngine_InputSystem_InputSettings_defaultButtonPressPoint); however, an individual control may [override](../api/UnityEngine.InputSystem.Controls.ButtonControl.html#UnityEngine_InputSystem_Controls_ButtonControl_pressPoint) this value), the Action goes from `Started` to `Performed`. One callback occurs on [`InputAction.performed`](../api/UnityEngine.InputSystem.InputAction.html#UnityEngine_InputSystem_InputAction_performed).
```

Comment:
```suggestion
2. If a Control then reaches or exceeds the button press threshold, the Action goes from `Started` to `Performed`. One callback occurs on [`InputAction.performed`](../api/UnityEngine.InputSystem.InputAction.html#UnityEngine_InputSystem_InputAction_performed). The default value of the button press threshold is defined in the [input settings](../api/UnityEngine.InputSystem.InputSettings.html#UnityEngine_InputSystem_InputSettings_defaultButtonPressPoint). However, an individual control can [override](../api/UnityEngine.InputSystem.Controls.ButtonControl.html#UnityEngine_InputSystem_Controls_ButtonControl_pressPoint) this value.
```

Reply:
Rephrased this to read easier - if you have a complex parenthetical in the middle of a sentence, moving the parenthetical to the end of the paragraph can make the whole paragraph easier to parse.

Reply:
Pushed